packages/logger/src/formatters/omitFields.js

```diff
@@ -1,7 +1,11 @@
 import _ from 'lodash';
 
-export function omitFields(fieldsToOmit = []) {
+export function omitFields(fieldsToOmit = [], omitFieldsPathRoots = []) {
```

Comment:
Absolutely backwards compatible so nothing currently used should fail

Reply:
but confusing api :/

Reply:
What do you mean exactly? 🤔

Reply:
that this is not so readable
`omitFields(['someField'], ['someOtherField'])`
the relation between them. and how it works
this is slightly better, but again, I think I wouldn't go for this feature, yet.
`omitFields(['someField'], {roots: ['root1', 'root2.nest1']})`